scipy/stats/_stats_py.py

```diff
@@ -6334,6 +6352,11 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
 
     By default, the p-value is determined by comparing the t-statistic of the
     observed data against a theoretical t-distribution.
+
+    (In the following, note that the argument `permutations` itself is
+    deprecated, but a nearly identical test may be performed by creating
+    an instance of `scipy.stats.PermutationMethod` with ``n_resamples=permutuations``
+    and passing it as the `method` argument.)
```

Comment:
We could just remove all of this temporarily. We will be writing an interactive notebook tutorial for permutation and Monte Carlo versions of the t-test (like https://scipy.github.io/devdocs/tutorial/stats/hypothesis_normaltest.html) shortly, and we will link to that from here.